vm/src/stdlib/ctypes/basics.rs

```diff
@@ -265,14 +261,21 @@ impl Buffer for PyCDataBuffer {
         &self.options
     }
 }
+pub struct RawBuffer {
```

Comment:
@coolreader18 is this what you have in mind?

Reply:
Yeah, it is.

Reply:
The safe expression of this type in Rust is `Box<[u8]>` which is exactly equivalent to this struct. Is this intended work to avoid somethiing like ownership check? I think there must be a better way.